settings.gradle.kts

```diff
@@ -1,16 +1,25 @@
-rootProject.name = "multiplatform-paging-root"
-
 enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 
 pluginManagement {
   repositories {
-    gradlePluginPortal()
     google()
+    gradlePluginPortal()
     mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+  }
+
+  plugins {
+    kotlin("multiplatform").version("1.7.10")
+    id("org.jetbrains.compose").version("1.2.2")
+    kotlin("android").version("1.7.10")
   }
```

Comment:
What is this for? This creates a split-brain problem with the version catalog.

Reply:
Not needed and copied over from a sample I was working with. Removing next commit